config/initializers/zeitwerk.rb

```diff
@@ -0,0 +1,39 @@
+Rails.autoloaders.main.inflector.inflect(
+  'ui' => 'UI',
+  'proxy_api' => 'ProxyAPI',
+  'sti' => 'STI',
+  'dhcp' => 'DHCP',
+  'dns' => 'DNS',
+  'tftp' => 'TFTP',
+  'external_ipam' => 'ExternalIPAM',
+  'bmc' => 'BMC',
+  'ui_notifications' => 'UINotifications',
+  'ipam' => 'IPAM',
+  'ssh' => 'SSH',
+  'ssh_provision' => 'SSHProvision',
+  'ssh_execution_provider' => 'SSHExecutionProvider',
+  'keep_current_request_id' => 'KeepCurrentRequestID',
+  'ec2' => 'EC2',
+  'aws' => 'AWS',
+  'gce' => 'GCE',
+  'aix' => 'AIX',
+  'nxos' => 'NXOS',
+  'vrp' => 'VRP',
+  'sso' => 'SSO',
+  'puppet_ca_certificate' => 'PuppetCACertificate',
+  'url_resolver' => 'URLResolver',
+  'ztp_record' => 'ZTPRecord',
+  'aaaa_record' => 'AAAARecord',
+  'ptr4_record' => 'PTR4Record',
+  'ptr6_record' => 'PTR6Record'
+)
+
+Rails.autoloaders.main.ignore(
```

Comment:
@adamruzicka, @ofedoren Do you guys know what's the difference between `Rails.autoloaders.main` and ``
Rails [docs](https://guides.rubyonrails.org/autoloading_and_reloading_constants.html#rails-autoloaders) doesn't say much, after while of searching still couldn't find the difference. From first look they return same object but then why there are two methods?

```ruby
Rails.autoloaders.main
=> #<Zeitwerk::Loader:0x00000000073af3e8
 @autoloaded_dirs=[],
 @autoloads={},
 @collapse_dirs=#<Set: {}>,
 @collapse_glob_patterns=#<Set: {}>,
 @eager_load_exclusions=#<Set: {}>,
 @eager_loaded=false,
 @ignored_glob_patterns=#<Set: {}>,
 @ignored_paths=#<Set: {}>,
 @inflector=ActiveSupport::Dependencies::ZeitwerkIntegration::Inflector,
 @initialized_at=2022-07-27 15:53:58.924070552 +0200,
 @lazy_subdirs={},
 @logger=nil,
 @mutex=#<Thread::Mutex:0x00000000073aefb0>,
 @mutex2=#<Thread::Mutex:0x00000000073aef88>,
 @on_load_callbacks={},
 @on_setup_callbacks=[],
 @on_unload_callbacks={},
 @reloading_enabled=false,
 @root_dirs={},
 @setup=false,
 @tag="rails.main",
 @to_unload={}>

 Rails.autoloaders.once
=> #<Zeitwerk::Loader:0x000000000af8bd80
 @autoloaded_dirs=[],
 @autoloads={},
 @collapse_dirs=#<Set: {}>,
 @collapse_glob_patterns=#<Set: {}>,
 @eager_load_exclusions=#<Set: {}>,
 @eager_loaded=false,
 @ignored_glob_patterns=#<Set: {}>,
 @ignored_paths=#<Set: {}>,
 @inflector=ActiveSupport::Dependencies::ZeitwerkIntegration::Inflector,
 @initialized_at=2022-07-27 15:54:07.749815776 +0200,
 @lazy_subdirs={},
 @logger=nil,
 @mutex=#<Thread::Mutex:0x000000000af8b6c8>,
 @mutex2=#<Thread::Mutex:0x000000000af8b628>,
 @on_load_callbacks={},
 @on_setup_callbacks=[],
 @on_unload_callbacks={},
 @reloading_enabled=false,
 @root_dirs={},
 @setup=false,
 @tag="rails.once",
 @to_unload={}>
```

Reply:
https://guides.rubyonrails.org/autoloading_and_reloading_constants.html#config-autoload-paths talks about which paths are managed by which autoloader